### PR TITLE
Added about Hritesh journey

### PR DESCRIPTION
**Pull request by hritesh94**
## Description

Added about myself a little to the journey and it looks good and  I am looking forward to it

### CODE REVIEW

**Review by hritesh94 — COMMENTED** [review]
Added my journey

**Review by hritesh94 — COMMENTED** [review]
Added about my journey

File: _layouts/sidebar.md [hunk 1]
```diff
@@ -16,6 +16,7 @@
     - [Ayushi George](../journeys/ayushigeorge.md)
     - [Ebuka Chuqz](../journeys/ebukachuqz.md)
     - [Habeeb B. Oyesile](../journeys/badbatunde.md)
+    - [Hritesh Goldar](../journeys/hritesh94.md)
```
Comment (vikasganiga05) [inline _layouts/sidebar.md:19]: Your link is automatically added by a github-action. Please remove it from here.

File: Data.json [hunk 2]
```diff
@@ -93,5 +93,10 @@
     "name": "Nikhil Dhariwal",
     "username": "404Nikhil",
     "avatar": "https://github.com/404Nikhil.png"
+  },
+  {
+    "name": "Hritesh Goldar",
+    "username": "hritesh94",
+    "avatar": "https://avatars.githubusercontent.com/u/89804314?s=400&v=4"
```
Comment (vikasganiga05) [inline Data.json:100]: My suggestion is to change your GitHub profile photo to same as in your twitter. And change the below link :point_down: 
```suggestion
    "avatar": "https://github.com/hritesh94.png" 
```
Or
you can directly use your twitter profile photo like :point_down: 
```suggestion
    "avatar": "https://pbs.twimg.com/profile_images/1572963702827257858/ETu4UReH_400x400.jpg" 
```

**Review by vikasganiga05 — CHANGES_REQUESTED** [review]
Thanks for adding your journey. I have suggested some changes please take a look at them.

File: _layouts/sidebar.md (hunk 1 above)
Comment (hritesh94) [reply _layouts/sidebar.md]: ok did it and i have created another pr,hope everything is ok now

**Review by vikasganiga05 — APPROVED** [review]
Looks good to me :+1: Thanks for adding your journey :pray:

**Pull request merged**